crates/holochain-conductor-runtime-ffi/scripts/build-debug.sh

```diff
@@ -4,7 +4,7 @@ cargo ndk --manifest-path ./Cargo.toml -t arm64-v8a \
  -o ../tauri-plugin-holochain-service/android/src/main/jniLibs \
   build
 
-cargo run --bin uniffi-bindgen generate \
+cargo run -p uniffi-bindgen --release generate \
```

Comment:
It says `debug` but is building in release mode. Is that intentional?

Reply:
Yeah I just want to build the library in debug mode, the bindgen cli tool I just want to use (and avoid rebuilding).